components/metrics/RealtimeHeader.js

```diff
@@ -33,7 +33,7 @@ export default function RealtimeHeader({ websites, data, websiteId, onSelect })
           <FormattedMessage id="label.realtime" defaultMessage="Realtime" />
         </div>
         <div>
-          <ActiveUsers className={styles.active} value={count} />
+          <ActiveUsers className={styles.active} value={count} websiteId={websiteId} />
```

Comment:
ActiveUsers should take either a `value` and display that or take a `websiteId` and run a query. It shouldn't have both. There is probably some other bug if it's not displaying correctly.

Reply:
it takes `websiteId` to display a specific website, otherwise it only takes from `value`. To handle it, I change `ActiveUsers.js` implementation

Reply:
The value is calculated on the previous page `RealtimeHeader.js`. The reason we don't want to run another query is because we already have the data.

Reply:
That's still work when selecting 'All websites', but sometimes want to know active users on specific website on realtime page.

Reply:
I also found out that active users is not displayed even though we are currently accessing it.

<img width="1415" alt="Screen Shot 2022-03-29 at 13 06 45" src="https://user-images.githubusercontent.com/19968375/160544369-32da75b5-4df1-44d3-a0eb-b1889479cc12.png">